bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/wizards/floating/IssueLicensePackPage.java

```diff
@@ -94,12 +84,22 @@ public void createControl(Composite parent) {
 		Dialog.applyDialogFont(composite);
 	}
 
-	@Override
-	public void setVisible(boolean visible) {
-		if (visible) {
-			init();
+	private void init() {
+		boolean render = license == null;
```

Comment:
how about to inline this variable?

Reply:
There is a temporal coupling here: after the second line of the function executes, the `license` is never null. We track `first render` this way - due to EMF Forms rendering specifics.